### ISSUE TITLE
sbnd/icaruscode data objects should move to sbnobj

### ISSUE THREAD

**Issue by cjbackhouse** [SBNSoftware/sbncode#9]

**Comment by PetrilloAtWork** [comment]
We need to identify which are the data products to be shared.
Currently `icaruscode` has CRT data products (`icaruscode/CRT/CRTProducts`) and trigger-related ones (`icaruscode/PMT/Trigger/Data`). `sbndcode` only has CRT data products.

I suppose that the first step is to move only the first ones. @tgrbrooks and Chris Hilgenberg should confirm they are equivalent...

**Comment by PetrilloAtWork** [comment]
Issue #10 created to track the creation of `sbnobj` repository.

**Comment by wesketchum** [comment]
OK, #2 complete I think, allowing this to proceed.

**Comment by tgrbrooks** [comment]
> We need to identify which are the data products to be shared.
> Currently `icaruscode` has CRT data products (`icaruscode/CRT/CRTProducts`) and trigger-related ones (`icaruscode/PMT/Trigger/Data`). `sbndcode` only has CRT data products.
> 
> I suppose that the first step is to move only the first ones. @tgrbrooks and @chilge should confirm they are equivalent...

I think our `CRTHit` and `CRTTrack` data products are the same but the `CRTData` products have diverged significantly between sbndcode and icaruscode

**Comment by wesketchum** [comment]
OK, I'm starting to work on some of this migrating of data products now ...

**Comment by wesketchum** [comment]
Initial branch at SBNSoftware/sbnobj#2

Have added in ICARUS PMT Trigger data objects, and CRT data objects across the board. For the CRT ones, I've made 'common' ones for CRTHit, CRTTrack, and CRTTzero, though I've also copied the existing versions in each namespace because I worried about backwards compatibility (those are in the _Legacy.hh file). We should consider if we need that/when we can drop it.

I've just allowed experiement-specific CRTData objects (with only namespaces differing to distinguish the names). We should also consider if we can merge those or what.
